src/preprocess/visitor.ts

```diff
@@ -200,22 +223,50 @@ export class Visitor {
   }
 
   visit$include(node: yaml.$include, path: string, env: Env): AnyButUndefined {
-    if (node.data.indexOf('.') > -1) {
-      const reduce: any = _.reduce; // HACK work around broken lodash typedefs
-      const lookupRes: any = reduce(node.data.split('.'), (result0: any, subKey: string) => {
+    const searchChunks = _parse$includeKeyChunks(node.data, path)
+    const lookupRes: any = _.reduce( // tslint:disable-line:no-any
```

Comment:
Any particular reason to use `_.reduce` here, rather than `searchChunks.reduce(...)`?

Reply:
old habits. Changing it.